tests/make_observations.py

```diff
@@ -227,4 +227,4 @@ def make_observations(
     args = parser.parse_args()
     orbits_df = pd.read_csv(args.in_file)
     observations = make_observations(orbits_df, orbit_type=args.orbit_type)
-    observations.to_hdf(args.out_file, key="data", mode="w", format="table")
+    observations.to_csv(args.out_file, index=False)
```

Comment:
I suspect this is the reason the test is failing. I've run into issues in the past with pandas serialization of floats and our tests are fairly aggressive at the moment. We may need to define the `float_format` argument here or reduce the test tolerance. If you set float_format to "%.16f", what happens?

Reply:
Okay, looks like `float_precision="round_trip"` does it when reading the test observation csv directly fixed the issue. Pandas is heavily optimized for read speed rather than precision for csv floats apparently.